.editorconfig

```diff
@@ -0,0 +1,14 @@
+# editorconfig.org
```

Comment:
I was thinking to share this config somehow too. Maybe via symlink or I don't know. Do you have any idea how could it done?

Reply:
No, I don't.

We can store it globally once in the user home directory. However, we will need to copy it to projects anyway.